solutions/core-landing-zone/mgmt-project/services.yaml

```diff
@@ -67,3 +67,17 @@ spec:
   resourceID: accesscontextmanager.googleapis.com
   projectRef:
     external: management-project-id # kpt-set: ${management-project-id}
+---
+# Access Context Manager API
```

Comment:
the comment should be updated to "Anthos API"

Reply:
Updated. Thanks!